update readme to correctly bootstrap AWS project (#820)
openfun/marsha: README.md

Create the shared state bucket where `Terraform` will keep all the information on your deployments so different developers/machines/CI processes can interact with them:

    $ ./bin/state apply

Initialize your `Terraform` config:

    $ make init

> The `make init` command will also create an [ECR](https://aws.amazon.com/ecr/) repository. Before going further you have to build and publish the lambda docker image. Unfortunately AWS doesn't allow to use a public image, yo have to host this one on a private ECR instance. Copy the output of the `init` command, you will use them in the next step.

Reviewer: ```suggestion
The `make init` command will also create an [ECR](https://aws.amazon.com/ecr/) repository. Before going further, you have to build and publish the lambda docker image. Unfortunately AWS doesn't allow to use a public image, so you have to host this one on a private ECR instance. Copy the output of the `init` command, you will use them in the next step.
```

#### Build and publish the lambda image

> For this step, we cooked a script for you responsible to build, tag and deploy images. All the scripts are runned from the marsha root directory.

Reviewer: ```suggestion
For this step, we cooked a script to help you build, tag and deploy images. All the scripts are run from the marsha root directory.
```

> 🔧 **Before you go further**, you need to create `./src/aws/env.d/lambda` and replace the relevant values with your own. The `ECR` url is available in `shared_resources` terraform output you copied before.

Reviewer: ```suggestion
🔧 **Before you go further**, you need to create `./src/aws/env.d/lambda` and replace the relevant values with your own. The `ECR` url is available in the `shared_resources` terraform output you copied earlier.
```

> You have to successively run this commands : 

Reviewer: ```suggestion
You have to successively run these commands : 
```

Build the image

    $ ./bin/lambda build

> Tag the iamge

Reviewer: ```suggestion
Tag the image:
```

    $ ./bin/lambda tag

> And then publish it

Reviewer: ```suggestion
And then publish it:
```

    $ ./bin/lambda publish

#### Apply all terraform plans

> Terraform is split in two parts. The main one, directly in `src/aws` can work on multiple [`Terraform` workspaces](https://www.terraform.io/docs/state/workspaces.html). You will use this feature if you want separated environment (development, staging, preprod and production). We also need some resources available across all workspaces. For this we have an other terraform in `src/aws/shared_resources`.

Reviewer: ```suggestion
Terraform is split in two parts. The main one, directly in `src/aws` can work on multiple [`Terraform` workspaces](https://www.terraform.io/docs/state/workspaces.html). You will use this feature if you want separate environments (development, staging, preprod and production). We also need some resources available across all workspaces. For this we have an other terraform in `src/aws/shared_resources`.
```
